src/main/java/Duke.java

```diff
@@ -42,17 +42,16 @@ public static void printGreeting() {
 
     public static void recordTasks(Task list[], int index, String command, String category) {
```

Comment:
Hi !! this method is too long, it would be better if you create another class for it :)